index.html

```diff
@@ -1,15 +1,91 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="es">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Segunda-Tarea</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Segunda-Tarea</title>
+	<title>Mi Tienda Online</title>
 </head>
 <body>
-  <header></header>
-  <main></main>
-  <footer></footer>
-</body>
+  <header class="header">
+    <div class="header__top">
+      <div class="header__top-top container">
+        <div class="header__logo">
+          <img src="image/logo.png" alt=>
+        </div>
```

Comment:
Revisando tu codigo, puedes lograr lo mismo con esto
```suggestion
        <img src="image/logo.png" alt="" class="header__logo">
```